util/common/src/main/java/org/hibernate/search/util/common/concurrent/ConcurrentReferenceHashMap.java

```diff
@@ -20,6 +20,7 @@
 import java.util.AbstractCollection;
 import java.util.AbstractMap;
```

Comment:
When you restore classes that were obviously copy/pasted from another project, please don't refactor them. It makes it very hard to track differences between projects.